src/vecs/client.py

```diff
@@ -83,6 +81,7 @@ def _supports_hnsw(self):
     def get_or_create_collection(
         self,
         name: str,
+        schema: str = "vecs",
```

Comment:
lets make these key word only arguments (after the *) on all method change signatures please

e.g.: list_collections, delete_collection etc